app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -23,7 +22,9 @@ class ProfileEditViewModel @Inject constructor(
   private val profileManagementController: ProfileManagementController
 ) : ObservableViewModel() {
   private lateinit var profileId: ProfileId
-  private lateinit var switch: Switch
+
+  private val _isAllowedDownloadAccess = MutableLiveData<Boolean>()
+  val isAllowedDownloadAccess: LiveData<Boolean> = _isAllowedDownloadAccess
```

Comment:
Public members of a class should have documentation. See https://developer.android.com/kotlin/style-guide#usage and https://developer.android.com/kotlin/style-guide#documentation.